test/core/end2end/no_server_test.cc

```diff
@@ -96,8 +96,9 @@ void run_test(bool wait_for_ready) {
   grpc_completion_queue_shutdown(cq);
   while (grpc_completion_queue_next(cq, gpr_inf_future(GPR_CLOCK_REALTIME),
                                     nullptr)
-             .type != GRPC_QUEUE_SHUTDOWN)
+             .type != GRPC_QUEUE_SHUTDOWN) {
     ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done